lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go

```diff
@@ -840,6 +867,11 @@ func deleteSubscriber(networkID, key string) error {
 }
 
 func loadAllStatesForIMSIs(networkID string, imsis []string) (map[string]state_types.StatesByID, error) {
+	imsiMap := make(map[string]int)
```

Comment:
Don't use hungarian notation https://docs.quantifiedcode.com/python-anti-patterns/readability/putting_type_information_in_a_variable_name.html -- instead, just call this e.g. `imsisExist`

Reply:
Use the literal declaration, and use the go-idiosyncrasy `struct{}` value
```suggestion
	imsisExist := &map[string]struct{}{}
```